src/test/integration/auxillaries.py

```diff
@@ -0,0 +1,41 @@
+import sys
+import json
+import argparse
+import unittest
+import requests
+from requests.auth import HTTPBasicAuth
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-i', help='path to input file', dest='inputfile')
+    namespace, args = parser.parse_known_args()
+    print(namespace, args)
+    return parser.parse_known_args()
+
+def getCardById(id, url, user, passw):
```

Comment:
Unlike the convention in Java/Groovy, we usually use lowercase and underscore to name a function/variable.
* https://www.python.org/dev/peps/pep-0008/?#function-and-variable-names
* https://stackoverflow.com/questions/8908760/should-i-use-camel-case-or-underscores-in-python